core/src/main/java/org/frankframework/util/Misc.java

```diff
@@ -293,6 +293,9 @@ public static String getDurationInMs(long value) {
 		return duration + "ms";
 	}
 
+	/**
+	 * @return 'age' in MS.
```

Comment:
```suggestion
	 * @return 'age' in milliseconds.
```
I don't like Microsoft 😉